src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/UniqueAccessCodeServiceImpl.java

```diff
@@ -375,13 +329,11 @@ private void validateUACCase(UAC uac, CollectionCase collectionCase) throws CTPE
     // https://collaborate2.ons.gov.uk/confluence/display/SDC/Auth.05+-+Unlinked+Authentication
 
     FormType uacFormType = FormType.valueOf(uac.getFormType());
-    CaseType uacCaseType = CaseType.valueOf(uac.getCaseType());
     CaseType caseCaseType = CaseType.valueOf(collectionCase.getCaseType());
-    Optional<LinkingCombination> linkCombo =
-        LinkingCombination.lookup(uacFormType, uacCaseType, caseCaseType);
+    Optional<LinkingCombination> linkCombo = LinkingCombination.lookup(uacFormType, caseCaseType);
```

Comment:
could be more succinct with orElseThrow(0

Reply:
Don't think orElseThrow will work, as we need to log that we are throwing an exception